.travis.yml

```diff
@@ -27,6 +27,8 @@ env:
     - LIBXML2_DEB="libxml2-dbg_2.9.1+dfsg1-3ubuntu4.9_amd64.deb"
     - POSTGRES_DEB="postgresql-9.6_9.6.3-1.pgdg12.4+1_amd64.deb"
     - ELASTICSEARCH_ARCHIVE="elasticsearch-2.4.5.tar.gz"
+    - ELASTICSEARCH6_ARCHIVE="elasticsearch-6.3.1.tar.gz"
+    - ELASTICSEARCH_HOST="127.0.0.1:9201"
```

Comment:
This appears unused, and were it used also should be named `ELASTICSEARCH6_HOST`, I believe.